changelog/unreleased/enhancement-app-provider-feedback

```diff
@@ -0,0 +1,5 @@
+Enhancement: Show errors when failing to open app in app provider
+
+The error message provided by wopi is now displayed to the user, giving some context on why it failedd to open a file.
```

Comment:
```suggestion
The error message provided by wopi is now displayed to the user, giving some context on why it failed to open a file.
```

Reply:
sorry.. I have a mac with the faulty keyboard and hadn't had time to fix it yet xD I tried cleaning but I guess I missed some words...